extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -187,4 +187,11 @@ public static final FlywayDataSourceRuntimeConfig defaultConfig() {
      */
     @ConfigItem
     public boolean validateMigrationNaming;
+
+    /**
+     * Ignore migrations during validate and repair according to a given list of patterns.
+     * Patterns are comma seperated.
+     */
+    @ConfigItem
+    public Optional<String> ignoreMigrationPatterns = Optional.empty();
```

Comment:
I believe you can use `List<String>` or `String[]` here (wrapped in an `Optional`)